### ISSUE TITLE
crack app

### ISSUE THREAD

**Issue by zZVanDucItZz** [luvyouso/CalendarGoogle2016#1]
run app
07-28 11:00:22.824 19343-19343/com.briswell.mycalendar I/art: Late-enabling -Xcheck:jni
07-28 11:00:22.873 19343-19349/com.briswell.mycalendar I/art: Debugger is no longer active
07-28 11:00:22.888 19343-19343/com.briswell.mycalendar W/System: ClassLoader referenced unknown path: /data/app/com.briswell.mycalendar-2/lib/arm
07-28 11:00:26.745 19343-19343/com.briswell.mycalendar W/System: ClassLoader referenced unknown path: /data/app/com.briswell.mycalendar-2/lib/arm
07-28 11:00:26.951 19343-19343/com.briswell.mycalendar D/ExtensionsFactory: No custom extensions.
07-28 11:00:27.073 19343-19343/com.briswell.mycalendar D/ExtensionsFactory: AllInOneMenuExtensions not found in properties file.
07-28 11:00:27.138 19343-19585/com.briswell.mycalendar W/AsyncQuery: Exception thrown during handling EVENT_ARG_QUERY
                                                                     java.lang.SecurityException: Permission Denial: opening provider com.android.providers.calendar.CalendarProvider2 from ProcessRecord{b89d1a8 19343:com.briswell.mycalendar/u0a246} (pid=19343, uid=10246) requires android.permission.READ_CALENDAR or android.permission.WRITE_CALENDAR
                                                                         at android.os.Parcel.readException(Parcel.java:1599)
                                                                         at android.os.Parcel.readException(Parcel.java:1552)
                                                                         at android.app.ActivityManagerProxy.getContentProvider(ActivityManagerNative.java:3550)
                                                                         at android.app.ActivityThread.acquireProvider(ActivityThread.java:4778)
                                                                         at android.app.ContextImpl$ApplicationContentResolver.acquireUnstableProvider(ContextImpl.java:2018)
                                                                         at android.content.ContentResolver.acquireUnstableProvider(ContentResolver.java:1468)
                                                                         at android.content.ContentResolver.query(ContentResolver.java:475)
                                                                         at android.content.ContentResolver.query(ContentResolver.java:434)
                                                                         at android.content.AsyncQueryHandler$WorkerHandler.handleMessage(AsyncQueryHandler.java:79)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                         at android.os.Looper.loop(Looper.java:148)
                                                                         at android.os.HandlerThread.run(HandlerThread.java:61)
07-28 11:00:27.151 19343-19343/com.briswell.mycalendar D/AllInOneActivity: not both, icicle:null intent:Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 cmp=com.briswell.mycalendar/.AllInOneActivity }
07-28 11:00:27.152 19343-19585/com.briswell.mycalendar W/AsyncQuery: Exception thrown during handling EVENT_ARG_QUERY
                                                                     java.lang.SecurityException: Permission Denial: opening provider com.android.providers.calendar.CalendarProvider2 from ProcessRecord{b89d1a8 19343:com.briswell.mycalendar/u0a246} (pid=19343, uid=10246) requires android.permission.READ_CALENDAR or android.permission.WRITE_CALENDAR
                                                                         at android.os.Parcel.readException(Parcel.java:1599)
                                                                         at android.os.Parcel.readException(Parcel.java:1552)
                                                                         at android.app.ActivityManagerProxy.getContentProvider(ActivityManagerNative.java:3550)
                                                                         at android.app.ActivityThread.acquireProvider(ActivityThread.java:4778)
                                                                         at android.app.ContextImpl$ApplicationContentResolver.acquireUnstableProvider(ContextImpl.java:2018)
                                                                         at android.content.ContentResolver.acquireUnstableProvider(ContentResolver.java:1468)
                                                                         at android.content.ContentResolver.query(ContentResolver.java:475)
                                                                         at android.content.ContentResolver.query(ContentResolver.java:434)
                                                                         at android.content.AsyncQueryHandler$WorkerHandler.handleMessage(AsyncQueryHandler.java:79)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                         at android.os.Looper.loop(Looper.java:148)
                                                                         at android.os.HandlerThread.run(HandlerThread.java:61)
07-28 11:00:27.282 19343-19343/com.briswell.mycalendar W/art: Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
07-28 11:00:27.302 19343-19343/com.briswell.mycalendar I/AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
07-28 11:00:27.529 19343-19343/com.briswell.mycalendar D/AllInOneActivity: Initializing to 1469678427151 for view 4
07-28 11:00:27.561 19343-19343/com.briswell.mycalendar D/AllInOneActivity: Adding handler with viewId 2131689607 and type 4
07-28 11:00:27.668 19343-19649/com.briswell.mycalendar E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #1
                                                                         Process: com.briswell.mycalendar, PID: 19343
                                                                         java.lang.RuntimeException: An error occurred while executing doInBackground()
                                                                             at android.os.AsyncTask$3.done(AsyncTask.java:309)
                                                                             at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
                                                                             at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
                                                                             at java.util.concurrent.FutureTask.run(FutureTask.java:242)
                                                                             at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
                                                                             at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
                                                                             at java.lang.Thread.run(Thread.java:818)
                                                                          Caused by: java.lang.SecurityException: Permission Denial: opening provider com.android.providers.calendar.CalendarProvider2 from ProcessRecord{b89d1a8 19343:com.briswell.mycalendar/u0a246} (pid=19343, uid=10246) requires android.permission.READ_CALENDAR or android.permission.WRITE_CALENDAR
                                                                             at android.os.Parcel.readException(Parcel.java:1599)
                                                                             at android.os.Parcel.readException(Parcel.java:1552)
                                                                             at android.app.ActivityManagerProxy.getContentProvider(ActivityManagerNative.java:3550)
                                                                             at android.app.ActivityThread.acquireProvider(ActivityThread.java:4778)
                                                                             at android.app.ContextImpl$ApplicationContentResolver.acquireUnstableProvider(ContextImpl.java:2018)
                                                                             at android.content.ContentResolver.acquireUnstableProvider(ContentResolver.java:1468)
                                                                             at android.content.ContentResolver.query(ContentResolver.java:475)
                                                                             at android.content.CursorLoader.loadInBackground(CursorLoader.java:64)
                                                                             at android.content.CursorLoader.loadInBackground(CursorLoader.java:56)
                                                                             at android.content.AsyncTaskLoader.onLoadInBackground(AsyncTaskLoader.java:312)
                                                                             at android.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:69)
                                                                             at android.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:66)
                                                                             at android.os.AsyncTask$2.call(AsyncTask.java:295)
                                                                             at java.util.concurrent.FutureTask.run(FutureTask.java:237)
                                                                             at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) 
                                                                             at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) 
                                                                             at java.lang.Thread.run(Thread.java:818) 
07-28 11:00:27.682 19343-19653/com.briswell.mycalendar D/OpenGLRenderer: Use EGL_SWAP_BEHAVIOR_PRESERVED: true
07-28 11:00:28.058 19343-19653/com.briswell.mycalendar I/Adreno-EGL: <qeglDrvAPI_eglInitialize:379>: QUALCOMM Build: 09/02/15, 76f806e, Ibddc658e36
07-28 11:00:28.066 19343-19653/com.briswell.mycalendar I/OpenGLRenderer: Initialized EGL, version 1.4
07-28 11:00:28.652 19343-19653/com.briswell.mycalendar E/Surface: getSlotFromBufferLocked: unknown buffer: 0xb39c57a0

**Comment by tunv11** [comment]
How many versions android, sir?

**Comment by zZVanDucItZz** [comment]
@luvyouso 
Android version 6.0 run on Nexus 5.

**Comment by tunv11** [comment]
Hi sir, run  genymotion ?, pls run device android.

**Comment by Ygauraw** [comment]
Thank you Dear,
I have build the project, Thanks to you :-)
Cheers!

```
On Monday, August 1, 2016 7:27 AM, LuvYouSo <notifications@github.com> wrote:
```

 Hi sir, run genymotion ?, pls run device android.—

**Comment by zZVanDucItZz** [comment]
@luvyouso 
I run device android.
My Device is Nexus 5 android version 6.0

**Comment by tunv11** [comment]
@zZVanDucItZz 
thanks sir,
project support < version 6.0.
i fix bug sersion 6.0
thanks a lot !

**Comment by tunv11** [comment]
@zZVanDucItZz 
ok, i fix bug permission for android version 6.0
thanks a lot.!
